conda-store-server/conda_store_server/_internal/server/views/api.py

```diff
@@ -128,6 +129,30 @@ def paginated_api_response(
     }
 
 
+def deprecated(sunset_date):
+    """
+    Decorator to add deprecation headers to a response. These will include
+        {
+            Deprecation: True
+            Sunset: <sunset_date>
+        }
+
+    :param sunset_date: the sunset date in HTTP-Date format, for example "Mon, 16 Feb 2025 23:59:59 UTC"
```

Comment:
Small nit: the majority of the repo is using numpydoc docstring format. In the future I'd really like to enable some docstring lint rules to make this more uniform. No worries about updating this; as we continue to update the code base we'll get more serious about ensuring docstring uniformity and completeness in the future.

As an aside, it would also be nice to have type annotations here. But again, most of the code base isn't annotated, so maybe we can leave this for the future?

Reply:
thanks for pointing this out! Lets update it now